src/Raven.Server/Documents/Queries/QueryMetadata.cs

```diff
@@ -567,8 +584,42 @@ void AddInclude(QueryExpression include, string path, ref List<string> listToAdd
                 CompareExchangeValueIncludes = compareExchangeValueIncludes?.ToArray();
         }
 
-        private static ExplanationField CreateExplanationField(MethodExpression expression)
+        private void AddToRevisionsInclude(RevisionIncludeField revisionIncludes, MethodExpression expression, BlittableJsonReaderObject parameters)
         {
+            foreach (var queryExpression in expression.Arguments)
+            {
+                switch (queryExpression)
+                {
+                    case FieldExpression fe:
+                        if (Query.From.Alias!= null)
+                            throw new InvalidOperationException($"Alias is not supported `include revisions(..)`.");
+
+                        revisionIncludes.AddRevision(fe.FieldValue);
+                        break;
+                    
+                    case ValueExpression ve:
+                        foreach ((object value, _)  in QueryBuilder.GetValues(Query, this, parameters, ve))
+                        {
+                            string? path = value.ToString();
```

Comment:
Remove the nullable annotation, it produces a warning

Reply:
it is strange, In Rider, it doesn't show any error whatsoever